indexing-service/src/main/java/org/apache/druid/indexing/overlord/TaskStorage.java

```diff
@@ -75,6 +75,13 @@
    */
   void removeLock(String taskid, TaskLock taskLock);
 
+  /**
+   * Remove the tasks created order than the given createdTime.
+   *
+   * @param timestamp timestamp to check the {@code created_date} of tasks
+   */
```

Comment:
I think the comment like below would be better:
```java
/**
   * Remove the tasks created order than the given timestamp.
   *
   * @param timestamp timestamp in milliseconds
   */
```

Reply:
Agree, please change "order" to "older" here as well.